### ISSUE TITLE
Using the Ignore array

### ISSUE THREAD

**Issue by RickAltman** [lorenzos/Minixed#20]
Hi, I love this little dynamo of a product! The last line of Configuration reads like this:

$ignore = array(); // Names of files and folders to not list (case-sensitive)

I presume it is a filter of sorts, so that I could instruct it to not show certain files. What is the syntax for using this, and can I use wildcards to, for example, filter out all JPG and PNG files?

**Comment by tim-elmer** [comment]
Rick,

Simpler than that; it's a case-insensitive comparison. Syntax should look like `$ignore = array("launch_codes.csv", "Executive Meeting Minutes.docx");` unless I'm mistaken.

**Comment by Dominiquini** [comment]
It would be really useful if we could use wildcards or regular expression to filter the results!

Thanks

**Comment by lorenzos** [comment]
You're right. I'll leave this open as a reminder, PRs are welcome anyway 😉
